### PR TITLE
Issue 8

### PR DESCRIPTION
**Pull request by tngTUDOR**
@cmutel can you take a look at this for solving #8

### CODE REVIEW

**Comment by tngTUDOR** [comment]
I didn't creat the png with the schema :(

File: bonsai_raw_data_schema.sql [hunk 1]
```diff
@@ -37,10 +37,21 @@ CREATE TABLE "temporal_extent" (
   "finishes" date
 );
 
+
+CREATE TABLE "datasource" (
+  "id" SERIAL PRIMARY KEY,
+  "label" TEXT,
+  "version" TEXT NOT NULL,
+  "source" TEXT,
```
Comment (cmutel) [inline bonsai_raw_data_schema.sql:50]: Please add text here indicating how to use this field

File: bonsai_raw_data_schema.sql [hunk 2]
```diff
@@ -37,10 +37,21 @@ CREATE TABLE "temporal_extent" (
...
+  "license_id" TEXT REFERENCES "license" ("identifier")
```
Comment (cmutel) [inline bonsai_raw_data_schema.sql]: To be consistent with other FKs, this should be to license.id

File: bonsai_raw_data_schema.sql [hunk 3]
```diff
@@ -37,10 +37,21 @@ CREATE TABLE "temporal_extent" (
...
+);
+
+
 CREATE TABLE "agent" (
```
Comment (cmutel) [inline bonsai_raw_data_schema.sql:56]: Can we add a FK from location as well?

File: bonsai_raw_data_schema.sql (hunk 1 above)
Comment (cmutel) [reply bonsai_raw_data_schema.sql:50]: Can we add a note here that this is a draft proposal until https://github.com/BONSAMURAIS/schema/issues/9 is resolved with more detail and a formal process?

**Review by cmutel — CHANGES_REQUESTED** [review]
Thanks! Just a few suggestions...

File: bonsai_raw_data_schema.sql (hunk 3 above)
Comment (tngTUDOR) [reply bonsai_raw_data_schema.sql:56]: one location ?

Comment (cmutel) [reply bonsai_raw_data_schema.sql:56]: Optional FK from location to datasource

**Review by cmutel — APPROVED** [review]

**Pull request merged**